inference-engine/src/legacy_api/src/ie_layer_validators.cpp

```diff
@@ -955,7 +955,11 @@ void DetectionOutputValidator::parseParams(CNNLayer* layer) {
 
     if (layer->CheckParamPresence("code_type")) {
         std::string _code_type = layer->GetParamAsString("code_type");
-        std::vector<std::string> code_types = {"caffe.PriorBoxParameter.CENTER_SIZE", "caffe.PriorBoxParameter.CORNER"};
+        for (auto& c: _code_type)
+        {
+            c = std::tolower(c);
```

Comment:
Why do you need convert it to lover case?

Reply:
When visitor api fetches string parameter it converts all letters to lowercase, so I needed to convert it to be able to compare them.